requirements.txt

```diff
@@ -1,3 +1,5 @@
+google-cloud-spanner==3.49.1
```

Comment:
This file has the most up to date dep versions to test against. The vertex dependency is only for testing, please add it to https://github.com/googleapis/langchain-google-spanner-python/blob/28c0addfeed9643239b6e16624f0c89ebae09c5b/pyproject.toml#L37 with the latest version. Pydantic is required by the library and needs to be added here https://github.com/googleapis/langchain-google-spanner-python/blob/28c0addfeed9643239b6e16624f0c89ebae09c5b/pyproject.toml#L11 with the range supported

Reply:
done